xml/ay_partitioning.xml

```diff
@@ -2566,6 +2566,104 @@ size=40G features='0' hwhandler='0' wp=rw
    </informaltable>
   </sect2>
 
+  <sect2 xml:id="ay-multidevice-btrfs">
+   <title>Multi-device Btrfs Configuration</title>
+
+   <para>
+    A <emphasis>Btrfs file system</emphasis> can be created on top of many
+    devices, offering similar features to a RAID device. With &ay;, a
+    <emphasis>multi-device Btrfs</emphasis> can be configured by specifying a
+    drive with the <literal>CT_BTRFS</literal> type. The <literal>device</literal>
+    property is used as an arbitrary name to identify each multi-device Btrfs.
+   </para>
+
+   <para>
+    As with RAID, you need to create all block devices first (e.g., partitions,
+    LVM logical volumes, etc.) and assign them to the Btrfs file system you want
+    to create over such block devices.
+   </para>
+
+   <para>
+    The following example shows a simple multi-device Btrfs configuration:
+   </para>
+
+   <example>
+    <title>Multi-device Btrfs configuration</title>
+      <screen>&lt;partitioning config:type="list"&gt;
```

Comment:
Just to let you know: You don't have to mask the `&gt;` at the end, it can also be a literal `>`. Only the `<` needs to be masked as `&lt;` inside `<screen>`.

It may or may not help/please your eyes. :wink:

Reply:
I did not know that! Thank you. I think it's more elegant to match them, though, and maybe less prone to incorrectly remembering which one you need to mask and which one you don't...?